webapp-django/crashstats/signature/static/signature/js/signature_tab_reports.js

```diff
@@ -115,5 +130,47 @@ SignatureReport.ReportsTab.prototype.buildUrl = function (params) {
 
     // Return the whole URL.
     return this.dataUrl + queryString;
+};
+
+SignatureReport.ReportsTab.prototype.onAjaxSuccess = function (contentElement, data) {
+    SignatureReport.Tab.prototype.onAjaxSuccess.apply(this, arguments);
+    var that = this;
+
+    $('.sort-header', contentElement).each(function() {
+        $(this).click(function(event) {
+            event.preventDefault();
+
+            var thisElt = $(this);
+
+            function removeFromArray(elt, arr) {
```

Comment:
What does eslint or jslint say about this? It's a function defined inside an iterator. It ought to be moved out to a more "global" scope.